addons/sales_team/models/crm_team.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models
+
+
+class CrmTeam(models.Model):
+    _name = "crm.team"
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    _description = "Sales Team"
+    _order = "name"
+    _period_number = 5
+
+    @api.model
+    def _get_default_team_id(self, user_id=None):
+        user_id = user_id or self.env.uid
+
+        team_id = self.env.context.get('default_team_id')
+        if not team_id:
+            team = self.search(['|', ('user_id', '=', user_id), ('member_ids', 'in', user_id)], limit=1)
+            team_id = team.id or self.env['ir.model.data'].xmlid_to_res_id('sales_team.team_sales_department')
```

Comment:
@WHPODOO I need only id then why should I use env.ref(And yes I show some comments where @WHPODOO told don't use ref when you need only id :))